### PR TITLE
change logs for 3.3.0

### PR DESCRIPTION
**Pull request by kdanryanlin**

### CODE REVIEW

**Comment by kdanryanlin** [comment]
@william-eth please review, thanks

File: change_log.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+# Ver.3.3.0
+## CHANGE LOGS
+
+### Released Date
+
+- Prodcution Env: 2022/11/03
+- Sandbox Env: 2022/10/3
```
Comment (william-eth) [inline change_log.md]: 日期格式統一一下

File: change_log.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+
+### New APIs
+
+- Part I. Admin
+    - Signature
+      - `POST` Create Stamp
+          - Using raw image to create a new stamp signature that can be used to fill in a signature field.
```
Comment (william-eth) [inline change_log.md]: Using a raw image ...

File: change_log.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+
+      - `DELETE` Delete Signatures
+          - To delete the specified signatures.
+
+### API Updated
+- Part I. Admin
+    - Task
+        1. `POST` Quick Create Task (Create by the Template)
+            - return an error `400934 template_need_sms_verify`,
+              if using the template in which signers should be verified with SMS OTP and not providing the signer's phone in request body.
```
Comment (william-eth) [inline change_log.md]: ... in the request body.

**Pull request merged**